config/samples/dataplane_v1beta1_openstackdataplane.yaml

```diff
@@ -55,6 +53,12 @@ spec:
         domainName: osptest.openstack.org
       preProvisioned: true
       nodeTemplate:
+        services:
+          - configure-network
+          - validate-network
+          - configure-os
+          - install-os
+          - run-os
```

Comment:
we will need to update: https://github.com/openstack-k8s-operators/install_yamls/blob/main/scripts/gen-edpm-kustomize.sh#L74-L78

Reply:
https://github.com/openstack-k8s-operators/install_yamls/pull/290 should fix that